packages/lit-dev-content/site/docs/components/lifecycle.md

```diff
@@ -330,6 +330,28 @@ async _loginClickHandler() {
 
 Also, when writing tests you can await the `updateComplete` Promise before making assertions about the component’s DOM.
 
+The `updateComplete` promise rejects if there's an unhandled error during the update cycle. For more information, see [Handling errors in the update cycle](#errors-in-the-update-cycle).
+
+### Handling errors in the update cycle {#errors-in-the-update-cycle}
+
+An uncaught exception in the update cycle causes the `updateComplete` promise to reject. If you're awaiting the `updateComplete` promise, you can handle these errors inline:
```

Comment:
It might good to note when there might be an unhandled error - Lit code doesn't have too many places to throw, but user code in render(), update(), etc., can throw and ReactiveElement will try to route that into the updateComplete promise.

So maybe:

```suggestion
An uncaught exception in the update cycle, which can be caused by exceptions in lifecycle methods like `render()` or `update()`, causes the `updateComplete` promise to reject. If you're awaiting the `updateComplete` promise, you can handle these errors inline:
```